src/snowflake/snowpark/open_telemetry_prototype.py

```diff
@@ -0,0 +1,76 @@
+import inspect
+from typing import Dict, Union
+
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor, ConsoleSpanExporter
+
+from opentelemetry import metrics
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader, ConsoleMetricExporter
+
+
+resource = Resource(attributes={
+    SERVICE_NAME: "snowpark-python-open-telemetry"
+})
+
+traceProvider = TracerProvider(resource=resource)
+processor = BatchSpanProcessor(ConsoleSpanExporter())
+traceProvider.add_span_processor(processor)
+
+trace.set_tracer_provider(traceProvider)
```

Comment:
You should not do this, you should just do `tracer = trace.get_tracer("df.filter.tracer")`, this will be done by the snowpark runtime, or by any other runtime use to run this library.